resources/views/layout/footer.blade.php

```diff
@@ -4,23 +4,23 @@
             <img class="w-24 md:w-32 shrink-0 mx-auto" src="{{ asset('identification/logo.svg') }}" alt="Blumilk logo"/>
         </div>
         <div class="flex justify-center items-center space-x-4 md:space-x-6">
-            <a href="https://clutch.co/profile/blumilk-0" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank">
+            <a href="https://clutch.co/profile/blumilk-0" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank" data-umami-event="outbound-link-click" data-umami-event-url="https://clutch.co/profile/blumilk-0">
                 <span class="sr-only">{{ __('alt.clutch') }}</span>
                 <x-icons.clutch></x-icons.clutch>
             </a>
-            <a href="https://github.com/blumilksoftware" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank">
+            <a href="https://github.com/blumilksoftware" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank" data-umami-event="outbound-link-click" data-umami-event-url="https://github.com/blumilksoftware">
                 <span class="sr-only">{{ __('alt.github') }}</span>
                 <i class="fa-brands fa-square-github text-xl lg:text-2xl"></i>
             </a>
-            <a href="https://linkedin.com/company/blumilksoftware" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank">
+            <a href="https://linkedin.com/company/blumilksoftware" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank" data-umami-event="outbound-link-click" data-umami-event-url="https://linkedin.com/company/blumilksoftware">
                 <span class="sr-only">{{ __('alt.linkedin') }}</span>
                 <i class="fa-brands fa-linkedin text-xl lg:text-2xl"></i>
             </a>
-            <a href="https://www.facebook.com/blumilksoftware/" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank">
+            <a href="https://www.facebook.com/blumilksoftware/" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank" data-umami-event="outbound-link-click" data-umami-event-url="https://www.facebook.com/blumilksoftware/">
                 <span class="sr-only">{{ __('alt.facebook') }}</span>
                 <i class="fa-brands fa-square-facebook text-xl lg:text-2xl"></i>
             </a>
-            <a href="https://www.youtube.com/@blumilksoftware" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank">
+            <a href="https://www.youtube.com/@blumilksoftware" class="text-gray-400 hover:text-gray-500 leading-none" target="_blank" data-umami-event="outbound-link-click" data-umami-event-url="https://www.youtube.com/@blumilksoftware">
```

Comment:
I think we should narrow down `data-umami-event` for better overview.
E.g. `data-umami-event=youtube-profile-link`

Reply:
I updated event names here:
https://github.com/blumilksoftware/website/pull/412/commits/c6c1c1b11479ccda281f5e4047a8707b32f7d6ae